types/miniprogram/tslint.json

```diff
@@ -0,0 +1,16 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please remove all these rules, we do not want any rules here for new packages

Reply:
Hi, @sheetalkamat, the definitely-files are generated from other codes which not totally meet the defaults rules.

Reply:
You would still need to fix the errors by removing all the rules here. We do not allow tslint to have any rules for new packages.